arango/request.py

```diff
@@ -26,6 +22,18 @@ def normalize_headers(
     return normalized_headers
 
 
+def generate_driver_header(driver_flags: Optional[DriverFlags] = None) -> str:
+    flags: str = ""
+    if driver_flags is not None:
+        for flag in driver_flags:
+            flags = flags + flag + ";"
+    name: str = "python-arango"
+    driver_version: str = version(name)
+    driver_header: str = name + "/" + driver_version + " (" + flags + ")"
+
+    return driver_header
```

Comment:
We have to make sure this doesn't break when we do `pip install -e .` for development mode.